.rpm/polkadot.spec

```diff
@@ -0,0 +1,50 @@
+%define __spec_install_post %{nil}
+%define __os_install_post %{_dbpath}/brp-compress
```

Comment:
I have written my own RPM spec files and have never seen either of these lines, so I recommend adding a comment explaining what they do.